### ISSUE TITLE
CI build failure

### ISSUE THREAD

**Issue by ribose-jeffreylau** [CalConnect/cc-transcription-systems#87]
https://github.com/CalConnect/csd-transcription-systems/runs/3228016066#step:5:252

```
<snip>
Downloading font "webcore" from https://download.opensuse.org/repositories/home:/alteratio:/Common/openSUSE_13.2/src/webcore-fonts-3.0-2.1.src.rpm
bundler: failed to load command: metanorma (/usr/local/bundle/bin/metanorma)
/usr/local/bundle/gems/down-5.2.2/lib/down/net_http.rb:148:in `rescue in open_uri': too many redirects (Down::TooManyRedirects)
	from /usr/local/bundle/gems/down-5.2.2/lib/down/net_http.rb:145:in `open_uri'
	from /usr/local/bundle/gems/down-5.2.2/lib/down/net_http.rb:93:in `download'
	from /usr/local/bundle/gems/down-5.2.2/lib/down/backend.rb:13:in `download'
<snip>
```

**Comment by ronaldtse** [comment]
We can cleanup the Gemfile so that only metanorma-cli and structured_csv remains. We also don't need the Gemfile.lock.

**Comment by ribose-jeffreylau** [comment]
@ronaldtse  Tried both but still same error.

**Comment by ribose-jeffreylau** [comment]
It magically fixed itself since 7b5f85aae2ae376987f9073549f0a64786cb88d2.  Closing.
